### PR TITLE
Android app pages

### PR DESCRIPTION
**Pull request by apt105**
See https://meshtastic-mt4g82lxn-apt105.vercel.app/docs/software/android/android-installation

### CODE REVIEW

File: website/docs/software/android/installation.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+id: android-installation
+title: Android application installation
+sidebar_label: Installation
+---
+Our Android application is available to download on Google Play.
+
+[<img alt="Download at https://play.google.com/store/apps/details?id=com.geeksville.mesh" src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png" style={{zoom:'35%'}} />](https://play.google.com/store/apps/details?id=com.geeksville.mesh&referrer=utm_source%3Dgithub-homepage)
```
Comment (jfirwin) [inline website/docs/software/android/installation.md:8]: Could this be centered on the page?

**Review by jfirwin — COMMENTED** [review]
Everything looks good to me, I don't have an android so I can't confirm the content, but I didn't see any glaring issues. Nicely done!

File: website/docs/software/android/installation.md (hunk 1 above)
Comment (apt105) [reply website/docs/software/android/installation.md]: You like giving me a challenge! It's easier to do in straight html than trying to work out centring something in markdown....

https://meshtastic-git-android-apt105.vercel.app/docs/software/android/android-installation

**Comment by sachaw** [comment]
Ready to merge?

**Comment by apt105** [comment]
I think it's ready unless anyone else has got any changes they want to see

**Pull request merged**